x-pack/plugins/canvas/types/webpack.d.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+declare module '*.png';
```

Comment:
What does this do?

Reply:
This allows TypeScript to understand that requiring a `.png` is a valid import.  While Webpack provides a string, in development it returns a bytestream... at which TypeScript balks.

https://github.com/Microsoft/TypeScript-React-Starter/issues/12#issuecomment-326370098

Reply:
That's probably a nice link to add as a comment here, like the `*.json` type in `x-pack/plugins/apm/typings/global_types.d.ts`